app.js

```diff
@@ -16,22 +17,18 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 app.use('/', indexRouter);
 
-// catch 404 and forward to error handler
-// app.use((req, res, next) => {
-//   next(createError(404));
-// });
-
 // error handler
 app.use((err, req, res, next) => {
   // return the error
   console.log({...err});
-  const errorMessage = err?.meta?.cause || err?.message || null;
+  const error = err || "An error occurred";
+  let statusCode = err.status || 500;
 
   if (err instanceof Prisma.NotFoundError) {
-    res.status(404).json({ error: errorMessage })
+    statusCode = 404;
```

Comment:
Prisma.NotFoundError c'est déprécié : 

`@deprecated
please don´t rely on type checks to this error anymore. This will become a PrismaClientKnownRequestError with code P2025 in the future major version of the client`